tests/test_toolkit/rag/test_example_workflow.py

```diff
@@ -0,0 +1,104 @@
+from typing import Annotated, Any, List
+
+from llama_index.core import VectorStoreIndex
+from llama_index.core.node_parser import SentenceSplitter
+from llama_index.core.query_engine import RetrieverQueryEngine
+from llama_index.core.response_synthesizers import get_response_synthesizer
+from llama_index.core.retrievers import VectorIndexRetriever
+from llama_index.llms.openai import OpenAI
+
+from notdiamond.toolkit.rag.evaluation import auto_optimize, evaluate
+from notdiamond.toolkit.rag.llms import get_embedding, get_llm
+from notdiamond.toolkit.rag.metrics import (
+    FactualCorrectness,
+    Faithfulness,
+    LLMContextRecall,
+    SemanticSimilarity,
+)
+from notdiamond.toolkit.rag.workflow import (
+    BaseNDRagWorkflow,
+    CategoricalValueOptions,
+    FloatValueRange,
+    IntValueRange,
+)
+
+
+class ExampleNDRagWorkflow(BaseNDRagWorkflow):
+    parameter_specs = {
+        "chunk_size": (Annotated[int, IntValueRange(1000, 2500, 500)], 1000),
+        "chunk_overlap": (Annotated[int, IntValueRange(50, 200, 25)], 100),
+        "top_k": (Annotated[int, IntValueRange(1, 20, 1)], 5),
+        "algo": (
```

Comment:
This param is not used in the `rag_workflow`. Just want to make sure that `CategoricalValueOptions` is working correctly